pkg/storage/chunk/client/alibaba/oss_object_client.go

```diff
@@ -35,10 +35,12 @@ type OssObjectClient struct {
 
 // OssConfig is config for the OSS Chunk Client.
 type OssConfig struct {
-	Bucket          string `yaml:"bucket"`
-	Endpoint        string `yaml:"endpoint"`
-	AccessKeyID     string `yaml:"access_key_id"`
-	SecretAccessKey string `yaml:"secret_access_key"`
+	Bucket               string `yaml:"bucket"`
+	Endpoint             string `yaml:"endpoint"`
+	AccessKeyID          string `yaml:"access_key_id"`
+	SecretAccessKey      string `yaml:"secret_access_key"`
+	ConnectionTimeoutSec int64  `yaml:"conn_timeout_sec"`
```

Comment:
can we use `time.Duration` instead and drop the `Sec` suffix?

Reply:
Good suggestion! but the client here only supports second unit, I am afraid if user configs it to less than 1 seconds will bring more problems.

Reply:
The smallest value is 1 second, it cannot be set to milliseconds according to the client's inner code:
```golang
client.Config.HTTPTimeout.ConnectTimeout = time.Second * time.Duration(connectTimeoutSec)
```